src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -28,7 +28,7 @@
 
 public class IntegrityCheck {
 
-    private final BibDatabaseContext bibDatabaseContext;
+    private static BibDatabaseContext bibDatabaseContext;
```

Comment:
What about passing the bibDatabaseContext in the constructor of the Field Checker?